doc/index.rst

```diff
@@ -1,41 +1,149 @@
-scikit-beam: Data Analysis Tools for X-ray Science
-==================================================
+:tocdepth: 2
 
-scikit-beam is a Python package providing tools for X-ray science.
-For a brief introduction to the ideas behind the package, you can read the
-:ref:`introductory notes <introduction>`.
+==================
+ Scikit-beam Core
+==================
 
-Users who prefer drag-and-drop software may prefer to use
-`vistrails <http://www.vistrails.org/index.php/Main_Page>`__, through which
-all the functionality of scikit-beam is also available.
-(Think LabView for X-ray image processing and data analysis.)
 
+The :ref:`vision<vision>` of `scikit-beam` is to provide simple
+functions useful for the X-ray, Neutron and Electron communities.  The
+primary goal of the scikit-beam project is to provide a centralized
+repository for algorithms that are used in these three scientific
+domains.  `scikit-beam` functions accept and return standard Python
+and numpy datatypes, so they integrate well with other packages from
+the scientific Python community.  Further, the modular design of
+scikit-beam allows its components to be easily reused in ways not
+envisioned by the authors.
```

Comment:
Looks good.  My only thought is that we do have classes that maintain state (for performance) so maybe "functions" should be changed to "functions/classes"?